indico/modules/events/papers/templates/management/cfp_actions/ended.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'events/papers/management/cfp_actions/_base.html' %}
+
+{% block label %}
+    {% trans %}The call for papers is closed{% endtrans %}
+{% endblock %}
+
+{% block text %}
+    {% set tz = event.timezone %}
+    {% trans date=cfp.end_dt|format_date(timezone=tz), time=cfp.end_dt|format_time(timezone=tz) -%}
+        Closed on <strong>{{ date }}</strong> at <strong>{{ time }}</strong> ({{ tz }}).
+    {%- endtrans %}
+{% endblock %}
+
+{% block buttons %}
+    {{ super() }}
+    <div class="group">
```

Comment:
I tested it and it works. Simply remove the wrapping `.group` div from all the cfp_actions files.